TWLight/resources/views.py

```diff
@@ -175,20 +175,30 @@ def get_context_data(self, **kwargs):
         context["apply"] = False
         context["has_open_apps"] = False
         context["has_auths"] = False
+        user = self.request.user
```

Comment:
same here; this can be streamlined (and more correct) by using the existing helper